integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,164 @@
+# Run Kobiton Automation Test on Travis CI
+This guide will demonstrate how to automation test on Kobiton devices with Travis CI.
+
+## Table of contents 
+  - [Prerequisites](#prerequisites)
+  - [1. Configure Travis CI with Github repository](#1-configure-travis-ci-with-github-repository)
+  - [2. Configure automation test script](#2-configure-automation-test-script)
+  - [3. Run automation script on Kobiton devices](#3-run-automation-script-on-kobiton-devices)
+  - [4. Get session information through REST API](#4-get-session-information-through-REST-API)
+  - [5. (Optional) Travis CI customization](#5-optional-travis-ci-customization)
+  - [6. Feedback](#6-feedback)
+
+## Prerequisites
+  - Kobiton account
+
+## 1. Configure Travis CI with Github repository
+
+  This part will guide you on how to sync Travis CI with a Github repository. If you already know how to do it, skip this step.
+   > For instruction on how to link Travis CI with a Github repository, follow [this guide.](https://docs.travis-ci.com/user/legacy-services-to-github-apps-migration-guide/)
+
+  This documentation will use Node.js as the default language to demonstrate the guide. Therefore, integrating Node.js plugin to Travis Ci is required.
+  > If you do not know how to integrate a Node JS app with Travis CI, follow [this tutorial.](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) 
+
+## 2. Configure automation test script
+  > You can visit [our blog](https://kobiton.com/blog/automation-web-appium-kobiton-nodejs/) for more details on setting up an automation test script.
```

Comment:
I think that the word 'script' should be 'script(s)' because I think one or **MORE** scripts might be executed.